explanation/cli_tools.rst

```diff
@@ -0,0 +1,71 @@
+Understanding juju vs jaas vs jimmctl
+=====================================
+
+This document aims to explain the differences between the various CLI tools you may encounter when using Juju and JAAS.
+
+=======
+Summary
+=======
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - 
+     - ``juju``
+     - ``jaas``
+     - ``jimmctl``
+   * - Purpose
+     - Used to interact with Juju/JIMM controllers.
+     - A plugin to add more commands to the Juju CLI.
+     - A tool for admins of the JIMM server.
```

Comment:
It feels like you can remove full stops from these single sentences in the table.